jwst/associations/lib/rules_level2_base.py

```diff
@@ -788,9 +786,11 @@ def __init__(self, association=None):
         # If an association is not provided, the check for original
         # exposure type is ignored.
         if association is None:
-            sources = lambda item: 'not imprint'
+            def sources(item): 
+                return 'not imprint'
         else:
-            sources = lambda item: association.original_exposure_type
+            def sources(item):
+                return association.original_exposure_type
```

Comment:
I assume these `def`s are equivalent to the lambdas, though I'm not sure what the lambdas are doing here anyway. I can revert this if it causes issues